crates/apub/src/lib.rs

```diff
@@ -82,7 +99,10 @@ fn check_apub_id_valid(apub_id: &Url, local_site_data: &LocalSiteData) -> Result
       .iter()
       .any(|i| domain.eq(&i.domain))
   {
-    return Err("Domain is not in allowlist");
+    return Err(LemmyErrorType::DomainNotInAllowList(format!(
+      "Domain '{}' is not in allow list",
+      domain
+    )))?;
```

Comment:
I would just set the domain as param for these, no need to format a message. That can be done by frontends/clients.